oat-sa/extension-tao-testqti
# Feature/inv 171/xml editor typing

__Related to task:__ https://oat-sa.atlassian.net/browse/INV-171
__Description:__ XML Editor - typing
__How to test:__ 
1. select a test and click the xml editor button
2. change xml test
3. click save
4. reload the page and make sure that the result is saved in the xml field
